public/src/models/banner.ts

```diff
@@ -52,4 +52,5 @@ export interface BannerTest extends Test {
   articlesViewedSettings?: ArticlesViewedSettings;
   controlProportionSettings?: ControlProportionSettings;
   deviceType?: DeviceType;
+  campaignName?: string;
```

Comment:
The field should be on the common `Test` type - https://github.com/guardian/support-admin-console/blob/main/public/src/components/channelManagement/helpers/shared.ts#L18
Also - unlike with scala, in TS you don't _have_ to define the field on each subtype as well, though there's nothing wrong with leaving it in.